waku/README.md

```diff
@@ -0,0 +1,287 @@
+# `waku`
+
+## Table of contents
+
+- [What is Waku?](#what-is-waku)
+- [Waku versioning](#waku-versioning)
+- [What does this package do?](#what-does-this-package-do)
+- [Waku package files](#waku-package-files)
+
+## What is Waku?
+
+Waku is a communication protocol for sending messages between Dapps. Waku is a fork of the [Ethereum Whisper subprotocol](https://github.com/ethereum/wiki/wiki/Whisper), although not directly compatible with Whisper, both Waku and Whisper subprotocols can communicate [via bridging](https://github.com/vacp2p/specs/blob/master/specs/waku/waku-1.md#backwards-compatibility).
+
+Waku was [created to solve scaling issues with Whisper](https://discuss.status.im/t/fixing-whisper-for-great-profit/1419) and [currently diverges](https://github.com/vacp2p/specs/blob/master/specs/waku/waku-1.md#differences-between-shh6-and-waku1) from Whisper in the following ways:
+
+- RLPx subprotocol is changed from `shh/6` to `waku/1`.
+- Light node capability is added.

```

Comment:
I believe light node was also available in whisper https://github.com/ethereum/go-ethereum/blob/510b6f90db406b697610fe0ff2eee66d173673b2/whisper/whisperv6/whisper.go#L291 , not sure if it's exactly the same, but the core concept seems to be (does not forward any envelope)

Reply:
Thanks for the feedback. Would that mean that [the specs](https://github.com/vacp2p/specs/blob/master/specs/waku/waku-1.md#differences-between-shh6-and-waku1) require clarification? As the specs say that light node capability is new.

I'll dig in the code a bit to see where the divergence is.

Reply:
oh, yes, looks like the capability is not new actually, nice catch, maybe worth updating the specs if that's the case

Reply:
Raised here https://github.com/vacp2p/specs/pull/117 . If you don't mind I will wait to see what the feedback is from this PR.

Reply:
So according to the feedback I got, the Vac team feel that making this change to the specs is not desirable. I will make an amendment to the docs to reflect the discrepancy between Waku light nodes being a direct fork of Whisper light nodes and simultaneously considered a new feature.

Reply:
Addressed with https://github.com/status-im/status-go/pull/1949/commits/493308ebf152a414d66ed962d321f8d690a459ab